tests/jax/test_fused_attn.py

```diff
@@ -338,42 +387,53 @@ def check_dqkv(primitive, reference, valid_len):
     pytest.param(AttnMaskType.PADDING_CAUSAL_MASK, id='PADDING_CAUSAL'),
 ])
 @pytest.mark.parametrize('qkv_layout', [
-    pytest.param(QKVLayout.BS3HD, id='qkvpacked'),
-    pytest.param(QKVLayout.BSHD_BS2HD, id='kvpacked'),
-    pytest.param(QKVLayout.BSHD_BSHD_BSHD, id='separate'),
+    pytest.param(QKVLayout.BS3HD, id='QKV_PACKED'),
+    pytest.param(QKVLayout.BSHD_BS2HD, id='KV_PACKED'),
+    pytest.param(QKVLayout.BSHD_BSHD_BSHD, id='SEPARATE'),
+])
+@pytest.mark.parametrize('dtype', [
+    pytest.param(jnp.bfloat16, id="BF16"),
+    pytest.param(jnp.float16, id="FP16")
+])
+@pytest.mark.parametrize('b, s_q, s_kv, h_q, h_kv, d',[
+    pytest.param(32,  128,  128, 16, 16, 64, id='SELF_512'),
+    pytest.param( 4, 2048, 2048, 12, 12, 64, id='SELF_ARB'),
+    pytest.param(32,  512,  128, 16, 16, 64, id='CROSS_512'),
+    pytest.param( 4, 2048, 1024, 12, 12, 64, id='CROSS_ARB'),
+    pytest.param(32,  128,  128, 12,  6, 64, id='GQA_512'),
+    pytest.param( 4, 2048, 2048, 12,  6, 64, id='GQA_ARB')
```

Comment:
I preferred the original naming so that we can regonize the failed tests at the first time instead of checking the source file especially for the CI.

Reply:
Yes, this is a temporary change here just to make the printouts a bit more concise/readable for me while debugging the CI failures. I will change them back to the original IDs before finalizing the PR.